src/pudl/transform/ferc1.py

```diff
@@ -227,40 +355,76 @@ def merge_instant_and_duration_tables_xbrl(
         instant = raw_xbrl_instant.drop(columns=drop_cols, errors="ignore")
         duration = raw_xbrl_duration.drop(columns=drop_cols, errors="ignore")
 
-        if instant.empty:
-            logger.debug(
-                f"{self.table_id.value}: No XBRL instant table found, returning the "
-                "duration table."
-            )
-            return duration
-        if duration.empty:
-            logger.debug(
-                f"{self.table_id.value}: No XBRL duration table found, returning "
-                "instant table."
-            )
-            return instant
-
         instant_axes = [
             col for col in raw_xbrl_instant.columns if col.endswith("_axis")
         ]
         duration_axes = [
             col for col in raw_xbrl_duration.columns if col.endswith("_axis")
         ]
-        if set(instant_axes) != set(duration_axes):
+        if (
+            bool(instant_axes)
+            & bool(duration_axes)
+            & (set(instant_axes) != set(duration_axes))
+        ):
```

Comment:
```suggestion
        if (
            (bool(instant_axes) | bool(duration_axes))
            & (set(instant_axes) != set(duration_axes))
        ):
```
bc we'd like to know if one is null but not the other... right?

Reply:
We only care if the index columns are the same on the two dataframes if we actually have two non-null dataframes. If one of them is empty, then we're fine.  There's a logging message below that lets us know if one of them is null.